src/content/docs/zh-cn/guides/integrations-guide/preact.mdx

```diff
@@ -120,6 +120,41 @@ export default defineConfig({
 目前，`compat` 选项只适用于 React 库，将代码以 ESM 的形式导出。如果在构建时出现错误，请尝试将该库添加到你的`vite.ssr.noExternal: ['the-react-library']` 的 `astro.config.mjs` 文件中。
 :::
 
+## 选项
+
+### 组合对个 JSX 框架
+
+当你在同一个项目中使用多个 JSX 框架（React、Preact、Solid）时，Astro 需要确定每个组件应该使用哪个 JSX 框架特定的转换。如果你只向你的项目中添加了一个 JSX 框架集成，那么就不需要额外的配置。
+
+使用 `include`（必填）和 `exclude`（可选）配置选项来指定哪些文件属于哪个框架。为你使用的每个框架提供一个文件数组或者文件夹。通配符可用于包含多个文件路径。
+
+我们建议将常见的框架组件放在同一个文件夹中（例如 `/components/react/` 和 `/components/solid/`），以便更容易地指定你的包含内容，但这不是必需的：
```

Comment:
```suggestion
我们建议将每个框架的组件放在同一个文件夹中（例如 `/components/react/` 和 `/components/solid/`），以便更容易地指定你的包含内容，但这不是必需的：
```

Original: `We recommend placing common framework components in the same folder...`

My understanding of `common` in this context is `shared by, coming from, or done by more than one`. Currently it's translated as `occurring, found, or done often; prevalent`.